App.js

```diff
@@ -21,25 +18,12 @@ export default function App() {
         <Stack.Screen name='Login' component={LoginView} />
         <Stack.Screen name='Dashboard' component={Dashboard} />
         <Stack.Screen name='CreateEventForm' component={CreateEventForm} />
+        <Stack.Screen name='FullEventCard' component={FullEventCard} />
       </Stack.Navigator>
     </NavigationContainer>
   );
 }
 
```

Comment:
Thanks for removing those comments. Those were unnecessary now that we're using the React Navigation.